js/main.js

```diff
@@ -22,6 +23,84 @@ $submitForm.addEventListener('submit', function (event) {
 
   data.nextEntryId++;
   data.entries.unshift(formData);
+
+  $ul.prepend(renderEntry(formData));
+  viewSwap('entries');
+
   $image.src = './images/placeholder-image-square.jpg';
   $submitForm.reset();
+  // toggleNoEntries();
+});
+
+// issue 2 code
+
+function renderEntry(entry) {
+  // generate and return a DOM tree for a single entry that matches the entries created in the unordered list
+  const $entryList = document.createElement('li');
+  $entryList.className = 'row';
+
+  const $entryDiv = document.createElement('div');
+  $entryDiv.className = 'column-half';
+
+  const $entryImg = document.createElement('img');
+  $entryImg.setAttribute('src', entry.photoURL);
+
+  const $otherDiv = document.createElement('div');
+  $otherDiv.className = 'column-half';
+
+  const $h1Entry = document.createElement('h1');
+  $h1Entry.textContent = entry.title;
+
+  const $pElement = document.createElement('p');
+  $pElement.textContent = entry.notes;
+
+  $entryList.appendChild($entryDiv);
+  $entryDiv.appendChild($entryImg);
+  $otherDiv.appendChild($h1Entry);
+  $otherDiv.appendChild($pElement);
+  $entryList.appendChild($otherDiv);
+
+  return $entryList;
+}
+
+const $ul = document.querySelector('.entry-list');
+document.addEventListener('DOMContentLoaded', function (event) {
+  for (let i = 0; i < data.entries.length; i++) {
+    const $data = renderEntry(data.entries[i]);
+    $ul.appendChild($data);
+  }
+});
```

Comment:
You're missing 2 more function calls in here

1. `toggleNoEntries` should be getting called so we can hide or show the text
2. `viewSwap` should be getting called so that the user loads the page on the same page they left off on